reference/catalogue.yaml

```diff
@@ -482,6 +482,99 @@ paths:
             '*/*':
               schema:
                 $ref: '#/components/schemas/Error'
+  /concepts:
+    get:
+      tags:
+        - Concepts
+      summary: /concepts
+      description: Returns a paginated list of concepts
+      operationId: getConcepts
+      parameters:
+        - name: query
+          in: query
+          description: Full-text search query
+          schema:
+            type: string
+        - name: identifiers.identifierType.id
```

Comment:
This should be `identifiers.identifierType`, but I'm not sure if that works as I'm having weird issues trying to test the filtering. I'm not getting consistent results. Not for this PR I know.